traffic_ops/app/lib/API/DeliveryServiceServer.pm

```diff
@@ -223,8 +240,19 @@ sub remove_server_from_ds {
 	my $ds_id  	 	= $self->param('dsId');
 	my $server_id	= $self->param('serverId');
 
-	if ( !&is_privileged($self) && !$self->is_delivery_service_assigned($ds_id) ) {
-		$self->forbidden("Forbidden. Delivery service not assigned to user.");
+	my $tenant_utils = Utils::Tenant->new($self);
+	if ( !&is_privileged($self) && !$tenant_utils->ignore_ds_users_table() && !$self->is_delivery_service_assigned($ds_id) ) {
```

Comment:
Probably yes, we need to decide if we merge only the 2 parameters (use-tenancy & ignore_ds_user_assignment) or the functions as well.
If we merge also the fucntions, lets do it in one "search replace" commit